### PR TITLE
Updated polish translation

### PR DESCRIPTION
**Pull request by mpasternak**

### CODE REVIEW

File: src/main/webapp/i18n/messages_pl.xml [hunk 1]
```diff
@@ -1952,265 +2114,281 @@
...
+			Libraries. The interface can be extensively modified through Manakin Aspects and XSL based Themes.
+			For more information visit
+			<a href="http://di.tamu.edu">http://di.tamu.edu</a> and
+			<a href="http://dspace.org">http://dspace.org</a>
+		</p>
+	</message>
+	<message key="xmlui.dri2xhtml.structural.contact-link">Kontakt z nami</message>
+	<message key="xmlui.dri2xhtml.structural.feedback-link">Wyślij uwagi</message>
+
+	<message key="xmlui.dri2xhtml.structural.head-subtitle">Repozytorium Centrum Otwartej Nauki</message>
```
Comment (bram-atmire) [inline src/main/webapp/i18n/messages_pl.xml]: This seems to be an institution specific header, maybe something more generic like Repozytorium DSpace would be more appropriate?

File: src/main/webapp/i18n/messages_pl.xml [hunk 2]
```diff
@@ -2273,266 +2451,55 @@
...
-    <message key="xmlui.ArtifactBrowser.AdvancedSearch.type_dateIssued">Data wydania</message>
-
-
-    <!-- Site Leve Recently Added Content -->
-    <message key="xmlui.ArtifactBrowser.SiteViewer.head_recent_submissions">Ostatnio dodane</message>
-
-    <message key="xmlui.ArtifactBrowser.AdvancedSearch.type_type_filter">Rodzaj</message>
-    <message key="xmlui.ArtifactBrowser.AdvancedSearch.type_publisher_filter">Wydawca</message>
-
+    <message key="xmlui.mirage2.itemSummaryView.MetaData">Metadata</message>
```
Comment (bram-atmire) [inline src/main/webapp/i18n/messages_pl.xml]: The mirage 2 keys are still in English

**Comment by mpasternak** [comment]
Hi @bram-atmire , thank you very much for taking the time for completing the review. 

Either I can't use the Python tools or I used them in the wrong way. Anyways, I will try to re-submit the pull request with a more readable diff. And, from what I've seen, I must have did something wrong, as I changed "Repozytorium Centrum Otwartej Nauki" to "Repozytorium DSpace" already so it looks like I have messed things up.  

Thanks again! I will re-submit the pull request soon.

**Comment by mpasternak** [comment]
Hi @bram-atmire , thank you very much for taking your time previously. This time, re-submitted the PR https://github.com/DSpace/dspace-xmlui-lang/pull/37 and I think you will be clearly able to see the improvement I did over this one. Thanks!

**Pull request closed without merging**